test/gtest/ucs/test_khash.cc

```diff
@@ -0,0 +1,29 @@
+/**
+* Copyright (C) NVIDIA Corporation. 2021.  ALL RIGHTS RESERVED.
+*/
+
+#include <common/test.h>
+
+#include <ucs/datastruct/khash.h>
+
+#include <string.h>
+
+KHASH_MAP_INIT_INT64(test_khash, size_t)
+
+class test_khash : public ucs::test {
+protected:
```

Comment:
why need protected?

Reply:
> why need protected?

leftover. originally was going to add internal template initialiser to allow check over several hash types, but removed it due to overcomplicated test

Reply:
👍 kis.